docker-compose-rule-core/src/main/java/com/palantir/docker/compose/execution/DefaultDockerCompose.java

```diff
@@ -141,6 +141,7 @@ private Version version() throws IOException, InterruptedException {
     private static String[] constructFullDockerComposeExecArguments(DockerComposeExecOption dockerComposeExecOption,
             String containerName, DockerComposeExecArgument dockerComposeExecArgument) {
         ImmutableList<String> fullArgs = new ImmutableList.Builder<String>().add("exec")
+                                                                            .add("-T")
```

Comment:
Could we put a tiny comment in here to capture how this is necessary for Windows?  I can imagine someone stumbling across this in future and wondering what a magic capital T does and why their unit tests seem to pass without it!!

https://docs.docker.com/compose/reference/exec/